pkgs/applications/office/spice-up/default.nix

```diff
@@ -0,0 +1,62 @@
+{ stdenv
+, fetchFromGitHub
+, intltool
+, autoreconfHook
+, pkgconfig
+, libqalculate
+, gtk3
+, granite
+, gnome3
+, json_glib
+, cmake
+, libgudev
+, libevdev
+, vala
+, wrapGAppsHook }:
+
+stdenv.mkDerivation rec {
+  name = "spice-up-${version}";
+  version = "1.2.1";
+
+  src = fetchFromGitHub {
+    owner = "Philip-Scott";
+    repo = "Spice-up";
+    rev = version;
+    sha256 = "0cbyhi6d99blv33183j6nakzcqxz5hqy9ijykiasbmdycfd5q0fh";
+  };
+  patches = [ ./0001-Remove-environment-dependant-CMake-configuration.patch ];
+
+  # Add extra packages to the VAPI path
+  # There is no environment variable, so we must edit the CMakeLists.txt
+  postPatch = ''
+    for f in $(find . -name CMakeLists.txt); do
+      substituteInPlace $f --replace '/vapi' '/vapi --vapidir ${granite}/share/vala/vapi --vapidir ${gnome3.libgee}/share/vala/vapi'
+    done
+  '';
+
+  nativeBuildInputs = [
+    intltool
+    pkgconfig
+    wrapGAppsHook
+    vala
+    cmake
+  ];
+  buildInputs = [
+    gtk3
+    granite
+    gnome3.libgee
+    json_glib
+    libgudev
+    libevdev
+    gnome3.gnome_themes_standard
+  ];
+
+  enableParallelBuilding = true;
```

Comment:
CMake builds in parallel by default: https://github.com/NixOS/nixpkgs/pull/32271